src/components/label-button.component.js

```diff
@@ -38,6 +38,31 @@ const styles = StyleSheet.create({
   },
 });
 
+export class InlineLabel extends Component {
```

Comment:
This might deserve its own file. We usually only create 1 component per file.

Reply:
that's fair. i mainly wanted to share the `getFontColorByBackground` function. is there an ideal location i should move that two so i can share it between components?

Reply:
Ooh, `getFontColorByBackground` might be a good thing to move to `src/utils` so we can access it from everywhere consistently!